tests/QueueTest.php

```diff
@@ -172,10 +172,10 @@ public function ensureCountIndexWithPrefixOfPrevious()
         $this->queue->ensureCountIndex(['type' => 1, 'boo' => -1], false);
         $this->queue->ensureCountIndex(['type' => 1], false);
 
-        $this->assertSame(2, count($this->collection->getIndexInfo()));
+        $this->assertSame(2, count($this->collection->listIndexes()));
```

Comment:
I'm not sure how it was working before, but the count of indexes should be 1. I think that's the issue w/ all of the ensure indexes tests